### ISSUE TITLE
Optional mappings

### ISSUE THREAD

**Issue by resolritter** [hkupty/trex.nvim#2]
I thought of submitting a PR doing the usual thing: putting the default remaps behind a global variable. However, I don't feel confident doing that because they seem to be all over the place.

https://github.com/Vigemus/trex.nvim/blob/master/lua/trex/init.lua#L71
https://github.com/Vigemus/trex.nvim/blob/master/lua/trex/fts/clojure.lua#L96
https://github.com/Vigemus/trex.nvim/blob/master/plugin/trex.vim#L10

I also don't want to meddle with the direction you pretend to take this plugin forward. Is it possible to organize all of those mappings into one place and/or tie them to a single variable? I'd like to set up my own. Cheers!

**Comment by hkupty** [comment]
Absolutely. I think this should be the right way. Trex was born almost by accident and I didn't devote much time to focus on it's structure - it was mostly on functionality I was getting out of iron.

I'll try to come up with a decent fix for it later today. :)

Best regards,
Henry
